python/lsst/pipe/supertask/config.py

```diff
@@ -47,6 +47,29 @@
 # ------------------------
 
 
+def InputDatasetField(doc, name, units, storageClass, check=None):
+    """Factory function to create Config class instances of InputDataSetConfig
+
+    This function servers as syntactic sugar for creating Configurable fields
+    which are InputDatasetConfigs. The nameing of this function violates the
+    normal convention of a lowercase first letter in the function name, as
+    this function is intended to sit in the same place as ConfigField classes,
+    and consistency in declaration syntax is important.
+
+    The input arguments for this class are a combination of the arguments for
+    ConfigField and InputDatasetConfig. The arguments doc and check come from
+    Config Field, while name, units, and storageClass come from
+    InputDatasetConfig.
```

Comment:
Please add a `Parameters` section and `Returns` section.

Reply:
I debated putting that information in or not, and opted to keep it more similar to other config class declarations in that file and elsewhere in the stack. However, I have no objection to putting them in either.

Reply:
Supertask should be using numpydoc so I vote for making this one correct rather than continuing to do it wrong.